pkg/util/utils.go

```diff
@@ -109,3 +110,17 @@ func MaybeChownDirRecursiveToMinikubeUser(dir string) error {
 func ParseKubernetesVersion(version string) (semver.Version, error) {
 	return semver.Make(version[1:])
 }
+
+// IsOnGCE determines whether minikube is currently running on GCE.
+func IsOnGCE() bool {
```

Comment:
how about moving this to where our WSL detection code is in the `driver` package ? 
https://github.com/medyagh/minikube/blob/8da592988da45acfc93cef2637e79a8bfa4ad570/pkg/minikube/driver/driver.go#L176

Reply:
it has absolutely nothing to do with drivers though

Reply:
we have tried to put any Machine related code in driver package (general one) I think it is more relevent to drivers than to util we try to keep util package as small as possible.

how about creating a new package called 
Platform or CloudEnvironment or something like that for both IsMicrosoftWSL(  and IsOnGCE